NAMESPACE

```diff
@@ -1,3 +1,5 @@
 # Generated by roxygen2: do not edit by hand
 
-import(rlang)
```

Comment:
So we decided to use `rlang::`. I was pushing a little bit to import rlang into package, but both approach are good for me, so we should use `rlang::`